packages/xml-views-validation/test/validators/attributes/unknown-attribute-key-spec.ts

```diff
@@ -0,0 +1,810 @@
+import { expect } from "chai";
+import { DocumentCstNode, parse } from "@xml-tools/parser";
+import { buildAst } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  generateModel,
+  expectExists,
+} from "@ui5-language-assistant/test-utils";
+import {
+  computeExpectedRange,
+  testValidationsScenario,
+} from "../../test-utils";
+import { validateUnknownAttributeKey } from "../../../src/validators/attributes/unknown-attribute-key";
+
+describe("the unknown attribute name validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
+    it("will detect an invalid attribute key in root class element", () => {
+      const xmlSnippet = `
+          <mvc:View
+            xmlns:mvc="sap.ui.core.mvc"
+            xmlns="sap.m"
+            🢂busy1🢀="true">
```

Comment:
could we use names that are more clearly wrong, e.g `busy_TYPO` / `busy_OOPS` / ...
- in general for this whole file.

Reply:
ok